unstructured/partition/utils/constants.py

```diff
@@ -39,6 +39,156 @@ class PartitionStrategy:
 
 TESSERACT_LANGUAGES_SPLITTER = "+"
 
+# source: https://tesseract-ocr.github.io/tessdoc/Data-Files-in-different-versions.html
+# All languages have been changed to lowercase and have been altered to remove dates and "(contrib)"
+# Ex: "Greek, Ancient (to 1453) (contrib)" -> "greek, ancient"
+# Where it seemed appropriate, languages have been split into multiple keys with the same value.
+# Ex: "greek, modern":"ell", "greek":"ell", "chinese - simplified":"chi_sim", "chinese":"chi_sim",
+# On tesseract-ocr.github.io, "Spanish" matches with both "spa_old" and "spa".
+# Here, it only matches with "spa" and "spanish - old":"spa_old" was added.
+TESSERACT_LANGUAGES_AND_CODES = {
+    "afrikaans": "afr",
+    "amharic": "amh",
+    "arabic": "ara",
+    "assamese": "asm",
+    "azerbaijani": "aze",
+    "azerbaijani - cyrilic": "aze_cyrl",
+    "belarusian": "bel",
+    "bengali": "ben",
+    "tibetan": "bod",
+    "bosnian": "bos",
+    "breton": "bre",
+    "bulgarian": "bul",
+    "catalan; Valencian": "cat",
+    "cebuano": "ceb",
+    "czech": "ces",
+    "chinese - simplified": "chi_sim",
+    "chinese": "chi_sim",
+    "chinese - traditional": "chi_tra",
+    "cherokee": "chr",
+    "corsican": "cos",
+    "welsh": "cym",
+    "danish": "dan",
+    "danish - fraktur": "dan_frak",
+    "german": "deu",
+    "german - fraktur (contrib)": "deu_frak",  # "contrib" not removed because it would repeat key
```

Comment:
why would removing contrib repeat a key? can't "german" and "german - fraktur" be structured in the same way as "danish" and "danish - fraktur"?

Reply:
Because there's another key with "german - fraktur" that has a different value. See line 90.